### PR TITLE
PLT-849: Kubescape Plugin

### PR DESCRIPTION
**Pull request by wcrum**

### CODE REVIEW

**Review by TylerGillson — CHANGES_REQUESTED** [review]

File: chart/validator/values.yaml [hunk 1]
```diff
@@ -426,6 +426,63 @@ plugins:
...
+        args:
+        - --health-probe-bind-address=:8081
+        - --leader-elect
+        containerSecurityContext:
+          allowPrivilegeEscalation: false
+          capabilities:
+            drop:
+            - ALL
+        image:
+          repository: quay.io/spectrocloud-labs/validator-plugin-oci
```
[the remark this answers is not in the document]
Comment (wcrum) [reply chart/validator/values.yaml]: fixed.

**Review by ahmad-ibra — CHANGES_REQUESTED** [review]
@wcrum are you able to address the vulnerability reported in this PR. It should be as simple as running `go get -u golang.org/x/net`

**Review by ahmad-ibra — COMMENTED** [review]
@wcrum are you able to confirm if the kubescape plugin will require any secrets? If it does, please set up the secret configuration in the `pluginSecrets` section at the bottom of the `chart/validator/values.yaml`

**Comment by wcrum** [comment]
> @wcrum are you able to confirm if the kubescape plugin will require any secrets? If it does, please set up the secret configuration in the `pluginSecrets` section at the bottom of the `chart/validator/values.yaml`

Kubescape plugin does not require any secrets as of right now, Kubescape itself does not have any secrets either.

**Review by ahmad-ibra — APPROVED** [review]
LGTM, thanks for getting this in!

**Review by TylerGillson — APPROVED** [review]

**Pull request merged**